apps/blog/src/posts/data-insights-on-PWAs/data-insights.md

```diff
@@ -0,0 +1,61 @@
+ * Launch handlers: In a PWA, the launch_handler field specifies how the PWA should be launched and navigated to. 
+ * Protocol handlers: In a PWA, protocol handlers are used to register the PWA to handle certain allowed protocols. A protocol is a string that allows clients to identify a resource on the web, such as http or mailto. When a PWA is installed, it can register to handle specific protocols, such as mailto or geo.
+ * Share target:  A share target is a feature in a Progressive Web App (PWA) that allows the app to receive shared content from other apps. This is done by adding a share_target member to the web app manifest, which specifies how the app should handle shared data. It allows for seamless integreation of the PWA with the native sharing capabilities of the device. 
+ * Shortcuts: Shortcuts in a Progressive Web App (PWA) allow users to quickly access specific features or pages within the app. These shortcuts can be added to the web app manifest and are displayed when a user long-presses or right-clicks on the app icon on their device.
+ * Widgets: Widgets are small applications that can be embedded within a larger application or operating system. They are designed to provide quick access to frequently used information or functionality. In the context of an app manifest, widgets are defined in the PWA manifest file using the widgets manifest member.
+
+## Insights from the data
+
+* It seems like  shortcuts are among the more popular features that developers like to use. Shortcuts are relatively easy to add, and are a quick way (or should a say, a shortcut), to add a feature that makes the PWA more app like.
+   
+* The number of PWAs with widgets might seem low, but it was actually a surprise to me as widgets are relatively a newer feature. Earlier this year, Facebook was one of the first big PWAs to add widgets to their manifest. Widgets are an easy way to increase engagement across different surfaces on an operating system. The uptick in widgets is a positive indicator of increasing awareness among developers.
+
+* There are many apps with detected protocol handlers, but upon a closer look, a large number of them were in fact empty arrays. It’s likely that the developers are using manifest templates that already contain protocol handlers but are not actually using them. 
+
+## PWA Spotlight
+
+While digging through the data, I stumbled upon this [music player PWA](https://xamuzik.com/) that had almost all the app capabilities. It’s available on multiple app stores including the Microsoft Store. The UX feels very app-like, and uses a lot of app capabilities that are relevant for this app and offer a rich native experience. Shout out to the developers! We love finding apps like these that exemplify what good PWAs are and how well PWAs can be integrated with the operating system. 
```

Comment:
I like the idea for this at the end, but it feels kinda tagged on at the end. A little more depth and maybe a screenshot might help.

Reply:
Yeah it started off as just a mention but became a whole paragraph..I don't want to put too much focus on it by adding more depth and stray away from the point of the blog. But I get what you're saying by "tagged on". I'll try to change it up a bit.